src/main/java/net/datafaker/providers/base/Internet.java

```diff
@@ -66,7 +66,36 @@ public String domainSuffix() {
         return resolve("internet.domain_suffix");
     }
 
+    /**
+     * Returns a web domain.
+     * 
+     * @return a domain in the form "www.example.com"
+     * @deprecated Since 2.0.0 
+     * @see #hostname()
+     * @see #uri()
+     */
+    @Deprecated(since="2.0.0",forRemoval = true) 
     public String url() {
+        return hostname();
+    }
+
+    /**
+     * Returns a string representing a URL, in the form "https://www.example.com/lorem/ipsun".
+     * 
+     * @return a URL, in the form "https://www.example.com/lorem/ipsun"
+     * @since 2.0.0
+     */
+    public String uri() {
+        return "https://" + hostname() + "/" + slug(faker.lorem().words(2), "/");
+    }
+
+    /**
+     * Returns a web domain.
+     * 
+     * @return a web domain in the form "www.example.com"
+     * @since 2.0.0
+     */
+    public String hostname() {
```

Comment:
A hostname can be anything, AFAIK. Eg "mars" could be a hostname, it's not something which has to start with www

Reply:
Yeah, sorry, I should have suggested something I guess. Maybe it's good to have a look at the RFCs, and see what is the terminology for url/uri/urn/hostname/etc

I don't mind having a hostname method, but I'm not sure what it should return:

localhost 
hello.localhost
example.com
www.example.com
demo.example.com
http://example.com
https://www.example.com

??

Reply:
Okay so for the purposes of this PR I'm going to go with web domain and have it return www.domain.tld :grinning:

Other stuff can be handled/added later if needed.

Reply:
Sounds fair!